### PR TITLE
set transparency to 0.95

### PR DESCRIPTION
**Pull request by retokromer**
- set transparency to 0.95 for better visibility
- correct "quit" also in commented out part

### CODE REVIEW

File: vrecord [hunk 1]
```diff
@@ -541,9 +541,8 @@ if [[ "${runtype}" = "audiopassthrough" ]] ; then
 #[f1][abcde1]hstack[abcdef1],\
 #[abcdef1][xx]overlay=200:10[vo]"
 #            ;;
-#        Q) exit 0 ;;
-#        q) exit 0 ;;
-#        *)  report -w "Error: Not a valid option, select a valid number or [q] to quit." ; return 1 ;;
+#        "quit") report -d "Bye then" ; exit 0 ;;
+#        *) report -w "Error: Not a valid option, select a valid number." ; return 1 ;;
```
Comment (dericed) [inline vrecord]: report is called before the function is defined.

Comment (retokromer) [inline vrecord]: report is defined at https://github.com/amiaopensource/vrecord/blob/f22cfc811b31a2ccc1adbe65c3c49c623d748715/vrecord#L253

Comment (dericed) [inline vrecord]: ok, you're right, this doesn't impact this pr, though there's still a report on line 65 that won't work properly

Comment (retokromer) [inline vrecord]: I’ll check that one (in another PR).

**Pull request merged**